API.md

```diff
@@ -50,7 +50,7 @@ See the endpoint descriptions for detailed usage of each field. This is an overv
 * Subject . (subject) The subject of the message. Limited in the API to 100 characters , but since replies are prefixed with 'Re: ' then in practice it is 96 characters.
 * Body . (body) Up to 10000 characters.
 * Survey . (survey). This is the uuid of the survey . It is mandatory when saving a message.
-* Collection Case . (collection_case) uuid of the collection case. Can be used as a filter option (cc). Used to send to the case service to inform it that a new message has been sent on the case.
+* Collection Case . (collection_case) uuid of the collection case. Can be used as a filter option (cc).
```

Comment:
I think you can remove this line completely.

Reply:
Removed the line and added a card to look at removing collection_case altogether from secure message after it's no longer being passed in.